style.css

```diff
@@ -562,3 +595,222 @@ button:disabled {
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .menu-list img {
+    margin-top: 6px;
+  }
+
+  .intro {
+    background-image: url(./img/Illustration-header.png);
+    height: 870px;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .intro-details p {
+    max-width: 566px;
+  }
+
+  .social-lg {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    width: 20px;
+    margin-right: 87px;
+  }
+
+  .social {
+    display: none;
+  }
+
+  .title-header {
+    flex-direction: row;
+    width: 100%;
+    padding-bottom: 0;
+    margin-bottom: 0;
+  }
+
+  .title-header h2 {
+    display: flex;
+    align-items: flex-end;
+    margin-right: 24px;
+    width: 315px;
+  }
+
+  .title-border {
+    min-width: 794px;
+  }
+
+  .work {
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    margin-top: 0;
+  }
+
+  .mobile-img {
+    display: none;
+  }
+
+  .desk-img {
+    max-width: 100%;
+    display: flex;
+  }
+
+  .work section {
+    max-width: 684px;
+  }
+
+  .recent-works {
+    margin-left: 141px;
+    margin-right: 143px;
+  }
+
+  .d-flex {
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .work-2 {
+    display: none;
+  }
+
+  .d-work {
+    background-size: cover;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .proj-work {
+    display: none;
+  }
+
+  .proj-work-1 {
+    width: 327px;
+    background-image: url(./img/Img-Placeholder-1.png);
+    background-repeat: no-repeat;
+    margin: 24px;
+    height: 435px;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .proj-btn {
+    margin: 0;
+  }
+
+  .proj-work-2 {
+    width: 327px;
+    background-image: url(./img/Img-Placeholder-2.png);
+    background-repeat: no-repeat;
+    margin: 24px;
+    height: 435px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .proj-work-3 {
+    height: 435px;
+    width: 327px;
+    background-image: url(./img/Img-Placeholder-3.png);
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    margin: 24px;
+  }
+
+  .langs-2 {
+    margin-bottom: 16px;
+  }
+
+  .proj-work-0 {
+    height: 435px;
+    width: 327px;
+    background-image: url(./img/Img-Placeholder-0.png);
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    margin: 24px;
+  }
+
+  .abt {
+    background-image: url(./img/illustration-about-me.svg);
+  }
+
+  .d-abt {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100%;
+  }
+
+  .abt-details p {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    min-width: 566px;
+    height: 148px;
+    line-height: 32px;
+  }
+
+  .technologies {
```

Comment:
- [ ] Align the technologies to the left for this section
![Screenshot from 2021-08-19 13-36-40](https://user-images.githubusercontent.com/65501535/130054770-2d8a3c75-c33d-47aa-90f8-9be99e1f12e6.png)

Original
![Screenshot from 2021-08-19 13-37-25](https://user-images.githubusercontent.com/65501535/130054865-9f923dce-e827-41ac-8888-d291f4bcd1aa.png)